app/views/simulator/edit.html.erb

```diff
@@ -27,6 +27,13 @@
             <li class="toolbarButton logixButton" id="save"><a>Save Online</a></li>
             <li class="toolbarButton logixButton" id="saveOffline"><a>Save Offline</a></li>
             <li class="toolbarButton logixButton" id="createOpenLocalPrompt"><a>Open Offline</a></li>
+            <li class="toolbarButton logixButton" id="createOpenOnlinePrompt">
+              <% if user_signed_in? %>
+                  <a href="<%= user_path(current_user) %>">Open Online</a>
```

Comment:
Do we really need this as we aare already redirecting to login path when not logged in

Reply:
Actually it is redirecting the user to open its own projects. Hence when the user is not logged in ,  then the user is re-directed to login . Please let me know your views on it.

Reply:
That means behavior is still the same except we will lose the page to which we need to redirect once the user is logged in

Reply:
So @tachyons Sir what should be done here?

Reply:
Just Don't show this menu when the user is not logged in

Reply:
Ok @tachyons Sir. Will fix this Soon.

Reply:
You may need to add condition befpre li block to avoid having an empty menu item